### DISCUSSION TITLE
Is it safe to remove VCF file after processing through seqr's pipeline?

### DISCUSSION THREAD

**Discussion by anjalijain22** [broadinstitute/seqr#4362] [Q&A] [answered]
Hi Seqr Team,

Hope you're doing well!

We have a local installation of seqr and are annotating our datasets on prem. We're running out of storage space so I'm trying to find files that can be removed. I am wondering if it is safe to remove a VCF from the` /mnt/data/seqr/data/input_vcfs/GRCh37` or `/mnt/data/seqr/data/input_vcfs/GRCh38` directory after we have processed it through seqr's pipeline and just retain the `.mt` file in the directory? I tried it for one project and the variant search functionality seems to work just fine but just wanted to confirm with you in case there are some downstream consequences I may not be aware of?

Many thanks,
Anjali

**Comment by bpblanken** [comment, marked as answer]
Hi @anjalijain22!

For the purposes of variant search functionality, removing the VCF is just fine.  However, we recommend you keep a copy of the raw VCF so you have it!

**Reply by anjalijain22** [reply]
Thank you very much for your prompt response! Yes, I am planning to move these VCF files to another space where we have more storage, they just wouldn't be accessible to seqr.

**Reply by bpblanken** [reply]
yep that sounds perfect!

**Reply by anjalijain22** [reply]
Hi again, quick follow up question - can the `.mt` files also be removed?

**Reply by bpblanken** [reply]
They `.mt` files produced by the pipeline may indeed be removed... the search functionality does not need them to work correctly.

**Reply by anjalijain22** [reply]
Amazing!!
I understand these are intermediate files, so is it okay to remove and not keep a back up or do you suggest I retain a copy of these elsewhere?

**Reply by anjalijain22** [reply]
> Amazing!! I understand these are intermediate files, so is it okay to remove and not keep a back up or do you suggest I retain a copy of these elsewhere?

Hi @bpblanken, sorry to bother you again! Just wanted to follow up on my previous question regarding deletion of the .mt files. Really appreciate your help on this!

**Reply by bpblanken** [reply]
Yes, it is ok to not keep a back up!  They are intermediate files that can be re-produced by running the pipeline over the same VCF.

**Reply by anjalijain22** [reply]
Perfect, thank you so much!!
